presto-spi/src/main/java/com/facebook/presto/spi/statistics/ColumnStatisticMetadata.java

```diff
@@ -83,7 +103,9 @@ public String toString()
         return "ColumnStatisticMetadata{" +
                 "columnName='" + columnName + '\'' +
                 ", statisticType=" + statisticType +
-                ", functionName=" + functionName +
+                ", functionName=" + function +
```

Comment:
nit: Should we change the column name to `function` or `functionSql`?

Reply:
Thanks for pointing this out. I made the json property and toString identifier name just `function`